packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -396,6 +396,7 @@ export class IndexedDbPersistence implements Persistence {
 
         // Attempt graceful shutdown (including releasing our owner lease), but
         // there's no guarantee it will complete.
+        // tslint:disable-next-line:no-floating-promises
         this.shutdown();
```

Comment:
This runs on the queue in the multi-tab branch. Should we change it here to also use `enqueueAndForget`?

This code is somewhat untested and I am a little afraid of its raciness if we don't enqueue it.

Reply:
I tried but we don't have access to the async queue here right now.  Maybe you should get around to merging multi-tab. 😛